src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -379,6 +379,11 @@ def test_llm_metrics_get_base_name(self) -> None:
         with pytest.raises(KeyError):
             metrics.get_base_name("hello1234")
 
+    def get_tokenizer(self):
```

Comment:
Is there a reason why we can't reuse `genai_perf.tokenizer.get_tokenizer`?

Reply:
generally I think we want to just test one module at a time to isolate it with minimal dependencies on other modules. However, we're already importing tokenizer stuff and it is essentially the same code, so I made the change you suggested.

Note that tests don't need the `add_bos_token=False` to work correctly. I just didn't want to update all of the expected values.

Reply:
Sg. Thanks for the explanation!